pkg/api/accesscontrol.go

```diff
@@ -344,6 +347,69 @@ func (hs *HTTPServer) declareFixedRoles() error {
 		Grants: []string{string(org.RoleEditor)},
 	}
 
+	var orgAnnotationReaderRole ac.RoleRegistration
+	var newDashboardAnnotationWriterRole ac.RoleRegistration
+
+	if hs.Features.IsEnabled(featuremgmt.FlagAnnotationPermissionUpdate) {
+		annotationsReaderRole = ac.RoleRegistration{
+			Role: ac.RoleDTO{
+				Name:        "fixed:annotations:reader",
+				DisplayName: "Annotation reader",
+				Description: "Read all annotations and tags",
+				Group:       "Annotations",
+				Permissions: []ac.Permission{
+					{Action: ac.ActionAnnotationsRead, Scope: dashboards.ScopeDashboardsAll},
+					{Action: ac.ActionAnnotationsRead, Scope: ac.ScopeAnnotationsTypeOrganization},
+				},
+			},
+			Grants: []string{string(org.RoleAdmin)},
```

Comment:
TODO: Viewer should be able to list all organization annotations, but not all dashboard annotations by default. Might need to split this into two roles.

Reply:
Done below, but might make more sense to have annotations dashboard reader instead.